openpype/plugins/publish/integrate.py

```diff
@@ -289,19 +296,17 @@ def register(self, instance, file_transactions, filtered_repres):
 
         template_name = self.get_template_name(instance)
 
-        subset, subset_writes = self.prepare_subset(instance)
+        subset, subset_writes = self.prepare_subset(instance, project_name)
         version, version_writes = self.prepare_version(instance, subset)
```

Comment:
Is this missing argument?

```suggestion
        version, version_writes = self.prepare_version(instance, subset, projet_name)
```

Reply:
Yep, didn't commit that.